conferences/2020/ux.json

```diff
@@ -425,6 +425,17 @@
     "cfpUrl": "https://uxdxconf.com/speak",
     "cfpEndDate": "2020-02-28"
   },
+  {
+    "name": "Oliver Disney",
```

Comment:
Please Change this to **Canvas Conference**

Reply:
Ok Done